sigma/pipelines/panther/crowdstrike_panther_pipeline.py

```diff
@@ -159,28 +159,11 @@ def crowdstrike_panther_pipeline():
                     logsource_windows_dns_query(),
                 ],
             ),
-            # ParentBaseFileName handling
-            ProcessingItem(
-                identifier="cs_parentbasefilename_fail_completepath",
-                transformation=DetectionItemFailureTransformation(
-                    "Only file name of parent image is available in CrowdStrike events."
-                ),
-                field_name_conditions=[
-                    IncludeFieldCondition(fields=["ParentBaseFileName"]),
-                ],
-                detection_item_conditions=[
-                    MatchStringCondition(
-                        cond="any",
-                        pattern="^\\*\\\\?[^\\\\]+$",
-                        negate=True,
-                    )
-                ],
-            ),
             ProcessingItem(
                 identifier="cs_parentbasefilename_executable_only",
                 transformation=ReplaceStringTransformation(
-                    regex="^\\*\\\\([^\\\\]+)$",
-                    replacement="\\1",
+                    regex=".*\\\\(.+)$",
```

Comment:
Sigma double \\ escape character makes this difficult to test in regular regex testers.  Does `C:\Program Files\Microsoft Monitoring Agent\Agent\MonitoringHost.exe` get replaced by `MonitoringHost.exe` ?

Reply:
almost
it is replaced by `\MonitoringHost.exe`

Reply:
It should not have the preceeding `\`